e2e/native-esm/__tests__/native-esm-wasm.test.js

```diff
@@ -54,3 +55,13 @@ test('imports from "data:application/wasm" URI with invalid encoding fail', asyn
     import('data:application/wasm;charset=utf-8,oops'),
   ).rejects.toThrow('Invalid data URI encoding: charset=utf-8');
 });
+
+test('supports wasm files that import js resources (wasm-bindgen)', async () => {
+  globalThis.alert = () => {};
```

Comment:
```suggestion
  globalThis.alert = jest.fn();
```

And delete line below.

(On mobile, so unable to make multi-line suggestions)

Reply:
Thanks! I committed your suggestion and also removed the next line: https://github.com/facebook/jest/pull/13608/commits/5d4a115227607c15798bee65ef476f7ed9c7fb40